src/eda.R

```diff
@@ -311,7 +319,11 @@ placement.transitions <- episodes %>% group_by(period_id, phase_number, placemen
   filter(!is.na(next_placement)) %>%
   mutate(transition_year = year(end + days(275)))
 
-placement.transitions.grouped <- placement.transitions %>% mutate(admission_age = transition_year - DOB) %>%
+transition_age <- function(end_of_placment, month_of_birth) {
+  floor(time_length(difftime(end_of_placment, as.Date(paste0(month_of_birth, "-01"), "%Y-%m-%d")), "years"))
+}
+
+placement.transitions.grouped <- placement.transitions %>% mutate(admission_age = transition_age(end,DOB)) %>%
```

Comment:
placement.transitions could be updated to include the birthday we have already imputed for the child?